README.md

```diff
@@ -10,32 +10,36 @@ This repository serves as a space for the Ansible Community Working Group to man
 
 ## Our Goals
 
-* Encourage transparency, collaboration, and accountability through tooling and lightweight project tracking and/or management
-* Provide guidance and advice to other working groups in the Ansible community
-* Facilitate the creation of new initiatives and Working Groups
+* Encourage transparency, collaboration, and accountability through tooling and lightweight project tracking and/or management.
+* Provide guidance and advice to other working groups in the Ansible community.
```

Comment:
What other working groups are there?

Also I'd prefer something along the lines of "Assist other working groups in the Ansible community for our mutual success." IMO "provide guidance and advice" sounds a little too much like we're governing other communities.

Reply:
agreed

Reply:
I'll remove the word "other".

Reply:
done, PTAL